snmp/datadog_checks/snmp/snmp.py

```diff
@@ -75,6 +76,8 @@ def __init__(self, *args, **kwargs):
 
         self._config = self._build_config(self.instance)
 
+        self._last_request_id = 0
```

Comment:
Shall we leverage the request-id of snmp protocol itself ?

![image](https://user-images.githubusercontent.com/49917914/95335869-30bea980-08b0-11eb-9ee9-4466ef64dc08.png)

Reply:
Doesn't seem to be what we want: `request-id` is a protocol detail, and only available on response, whereas this ID would be shared for a couple of commands that are in the same "fetch context".

I updated the PR to use the term "fetch ID" rather than "request ID".